ConanProfilesManager.cs

```diff
@@ -97,6 +97,7 @@ public void GenerateProfilesForProject(Project project)
                                 IVCRulePropertyStorage generalRule = vcConfig.Rules.Item("ConfigurationGeneral") as IVCRulePropertyStorage;
                                 string languageStandard = generalRule == null ? null : generalRule.GetEvaluatedPropertyValue("LanguageStandard");
                                 string cppStd = getConanCppstd(languageStandard);
+                                string runtime = vcConfig.Evaluate("$(RuntimeLibrary)").ToString().Contains("DLL") ? "dynamic" : "static";

```

Comment:
I think this is evaluating always to static as `vcConfig.Evaluate("$(RuntimeLibrary)")` is not a valid way to get the runtime